test/libsolidity/util/BytesUtils.cpp

```diff
@@ -82,22 +119,22 @@ bytes BytesUtils::convertString(string const& _literal)
 	}
 }
 
-string BytesUtils::formatUnsigned(bytes const& _bytes) const
+string BytesUtils::formatUnsigned(bytes const& _bytes)
 {
 	stringstream os;
 
-	if (*_bytes.begin() & 0x80)
-		os << u2s(fromBigEndian<u256>(_bytes));
-	else
-		os << fromBigEndian<u256>(_bytes);
+	soltestAssert(!_bytes.empty(), "");
```

Comment:
@chriseth Fixed https://github.com/ethereum/solidity/pull/7048#discussion_r301515345 here. Please also see the check below.